src/printing.py

```diff
@@ -11,7 +11,9 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print('x is %s, y is %s, z is %s' % (x, y, z))
 
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print('x is {x}, y is {y}, z is {z}'.format(x=x, y=y, z=z))
+# Finally, print the same thing using an f-string
+print(f'x is {x:<>5}, y is {y:*^20.3}, z is {z}')
```

Comment:
check this line again, it looks very weird when I run it.

Reply:
I was just playing around with the formatting to see what interesting results I could get on this one..